spec/models/comment_spec.rb

```diff
@@ -15,4 +15,14 @@
       Comment.create(commentable: issue, content: 'rspec content', user: user)
     end.to change { Subscription.count }.by(1)
   end
+
+  describe '#mentions' do
```

Comment:
Why does this need to be a public method? I don't see it being used anywhere except `subscribe_mentions` which itself is private?

Reply:
But wait, this feels to hacky, and I think it makes sense being able to call `comment.mentions`.
Reverted my last change and only made `subscribe_mentions` private